ansible_collections/arista/avd/molecule/eos_validate_state/reports/FABRIC-state.md

```diff
@@ -61,7 +60,7 @@
 | NTP | 19 | 0 | 0 | 0 |
 | OSPF | 4 | 0 | 0 | 0 |
 | Profiles | 4 | 0 | 0 | 0 |
-| Routing | 18 | 0 | 0 | 0 |
+| Routing | 6 | 0 | 0 | 0 |
```

Comment:
so some tests are changing categories -> are we happy with these changes? here missing 12 tests

Reply:
Yes, BFD tests from ANTA have been moved from `Routing, Generic` to `BFD`.